unified-runtime/source/adapters/level_zero/v2/event.cpp

```diff
@@ -232,6 +237,14 @@ ur_result_t urEventRelease(ur_event_handle_t hEvent) try {
 ur_result_t urEventWait(uint32_t numEvents,
                         const ur_event_handle_t *phEventWaitList) try {
   for (uint32_t i = 0; i < numEvents; ++i) {
+    if (!phEventWaitList[i]->getIsEventInUse()) {
+      // TODO: This is a workaround for the underlying inconsistency
```

Comment:
So what's the state of regular and counter-based events by default right now? Is it that regular events are not signaled? If so, could we just signal them manually?